src/java.base/share/classes/java/io/BufferedOutputStream.java

```diff
@@ -89,13 +85,10 @@ private BufferedOutputStream(OutputStream out, int initialSize, int maxSize) {
             throw new IllegalArgumentException("Buffer size <= 0");
         }
 
+        this.buf = new byte[initialSize]; // resizable if initialSize < maxSize
```

Comment:
Something has got messed up here, it creates the byte[] twice now.

Reply:
Yeah, @liach caught that above: fixing.